man/systemd.socket.xml

```diff
@@ -935,6 +935,20 @@
         <xi:include href="version-info.xml" xpointer="v255"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DeferTrigger=</varname></term>
+
+        <listitem><para>Takes a boolean argument. May only be used when <varname>Accept=no</varname>.
```

Comment:
BTW, I think it should not be restricted to `Accept=no`, potentially.
Maybe better to reword to `Currently it is supported when Accept=no.` ?

Reply:
It's OK to relax this later. But I don't want to proactively word it in a way that suggests this will be.

Reply:
OK, feel free to keep it.